### PR TITLE
Also change logs's ownership

### PR DESCRIPTION
**Pull request by yandy**
when we mount logs, we also need change ownership of dir 'logs'

### CODE REVIEW

**Comment by yosifkit** [comment]
Can swap the spaces to match surrounding tabs and apply it to 1.3 and 1.5 as well?

**Comment by yandy** [comment]
@yosifkit Hi, I've swapped the spaces to match surrounding tabs and applied it to 1.3 & 1.5. Please check it.

File: 1.4/docker-entrypoint.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ fi
 if [ "$1" = 'elasticsearch' ]; then
 	# Change the ownership of /usr/share/elasticsearch/data to elasticsearch
 	chown -R elasticsearch:elasticsearch /usr/share/elasticsearch/data
+  chown -R elasticsearch:elasticsearch /usr/share/elasticsearch/logs
```
Comment (yosifkit) [inline 1.4/docker-entrypoint.sh]: Still spaces here :wink:, other than that, LGTM

**Comment by tianon** [comment]
This is actually a dupe of the discussion over in #18.

File: 1.4/docker-entrypoint.sh (hunk 1 above)
Comment (yandy) [inline 1.4/docker-entrypoint.sh]: my mistake:-)

**Comment by mi-hol** [comment]
@tianon With 'Closed #18 via #150.' I'd guess this can be closed too

**Comment by yosifkit** [comment]
@mi-hol thanks.

**Pull request closed without merging**